.devcontainer/generate_ulid_func.sql

```diff
@@ -0,0 +1,86 @@
+ALTER DATABASE renku WITH is_template TRUE;
```

Comment:
question: I've used this before for testing and it's very useful. it's also useful in migrations, but since  I only needed a one-off back then, I didn't add it. But I wonder if we should just add this through a migration to our database? It's also useful when e.g. debugging in a PR deployment and testing migrations there.